feature/experimental/p4rt/wbb/example/main.go

```diff
@@ -0,0 +1,375 @@
+/*
+ * Copyright (c) 2022 Cisco Systems, Inc. and its affiliates
+ * All rights reserved.
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package main
+
+import (
+	"flag"
+	"github.com/cisco-open/go-p4/p4rt_client"
+	"github.com/cisco-open/go-p4/utils"
+	"github.com/golang/glog"
+	"github.com/openconfig/featureprofiles/feature/experimental/p4rt/wbb"
+	p4_v1 "github.com/p4lang/p4runtime/go/p4/v1"
+	codes "google.golang.org/grpc/codes"
+	"net"
+	"os"
+	"time"
+)
+
+// Command line args
+var (
+	serverIP   = flag.String("server_ip", "192.168.0.1", "P4RT Server IP")
```

Comment:
Same here please use IPs mentioned in https://github.com/openconfig/featureprofiles/blob/main/CONTRIBUTING.md

Reply:
This part is from https://github.com/openconfig/featureprofiles/pull/490. I will work with Reda to resolve this. Thanks.